README.md

```diff
@@ -1,3 +1,5 @@
+<img src="./Functions.Fox.Python.png" width = "320" alt="Functions Fox Header Image">
```

Comment:
I believe it makes more structural sense to have this in the docs folder rather than the root of the project. You can push again with the change and the CI should run fine this time.

Reply:
Could you also rebase onto the latest `dev` branch before pushing? It would keep the commit history cleaner. Thanks!

Reply:
Rebased and moved the image. Let me know if that worked correctly. I just hit the big rebase button in Github desktop and let Nat Friedman take the wheel.